include/mpu_wrappers.h

```diff
@@ -177,7 +177,6 @@
     #define PRIVILEGED_FUNCTION
     #define PRIVILEGED_DATA
     #define FREERTOS_SYSTEM_CALL
-    #define portUSING_MPU_WRAPPERS    0

```

Comment:
why isn't that also in  FreeRTOS.h similar to configSTACK_ALLOCATION_FROM_SEPARATE_HEAP?